torch/testing/_internal/opinfo/definitions/linalg.py

```diff
@@ -1276,6 +1276,9 @@ def make_input():
                 "test_fn_fwgrad_bwgrad",
                 dtypes=(torch.complex128,),
             ),
+            DecorateInfo(
+                toleranceOverride({torch.float64: tol(atol=1e-4, rtol=1e-4)})
+            ),
```

Comment:
Hmm, this should nothing to do with multigpu, shouldn't it?
Also, would it be possible to limit it to just a particular test? I.e. using something along the following lines
```suggestion
            DecorateInfo(
                toleranceOverride({torch.float64: tol(atol=1e-4, rtol=1e-4)}),
                "TestGradients",
                "test_fn_gradgard",
                device_type="cuda",
            ),
```

Reply:
Yes, this at first sight has nothing to do with multiGPU, but this is a test that is fairly flaky and it was activated last week, so I'm fine with running it with more lax tolerances.